OnFailedProcessMessage/handler.ts

```diff
@@ -23,21 +40,61 @@ type Handler = (c: Context, i: unknown) => Promise<void>;
  */
 export const getOnFailedProcessMessageHandler = ({
   lMessageStatusModel,
+  lMessageModel,
   telemetryClient
 }: IOnFailedProcessMessageHandlerInput): Handler =>
   withJsonInput(
-    withDecodedInput(CreatedMessageEvent, async (_, { messageId }) => {
-      await getMessageStatusUpdater(
-        lMessageStatusModel,
-        messageId
-      )(MessageStatusValueEnum.FAILED)();
-
-      telemetryClient.trackEvent({
-        name: "api.messages.create.failedprocessing",
-        properties: {
-          messageId
-        },
-        tagOverrides: { samplingEnabled: "false" }
-      });
-    })
+    withDecodedInput(CreatedMessageEvent, async (_, { messageId }) =>
+      pipe(
+        // Query for message with input messageId in order to retrieve the fiscalCode
+        lMessageModel.getQueryIterator({
+          parameters: [{ name: "@messageId", value: messageId }],
+          query: `SELECT * FROM m WHERE m.id = @messageId`
```

Comment:
Doesn't change the query cost, only to add more readability
```suggestion
          query: `SELECT TOP 1 * FROM m WHERE m.id = @messageId`
```